yb-voyager/src/tgtdb/yugabytedb_test.go

```diff
@@ -290,3 +291,189 @@ func TestGetPrimaryKeyConstraintNames(t *testing.T) {
 		testutils.AssertEqualStringSlices(t, tt.expectedPKNames, pkNames)
 	}
 }
+
+func TestCollectPgStatStatements_BasicSelectQueries(t *testing.T) {
```

Comment:
There seems to be a lot of duplication of code between the two functions 
TestCollectPgStatStatements_BasicSelectQueries and TestCollectPgStatStatements_InsertUpdateDeleteQueries

Can we merge the common logic of the two functions?
Something like:

```
func TestCollectPgStatStatements(t *testing.T) {
    testCases := []struct {
        name        string
        testQueries map[string]QueryTest
    }{
        {"BasicSelectQueries", selectQueries},
        {"InsertUpdateDeleteQueries", dmlQueries},
    }
    
    for _, tc := range testCases {
        t.Run(tc.name, func(t *testing.T) {
            // Common test logic
        })
    }
}
```

Reply:
moved the code to a helper function, take a look.